### PR TITLE
Fix README.md display on PyPI

### PR DESCRIPTION
**Pull request by westurner**
## What this fixes and how it's fixed

- The README.md render on PyPI: https://pypi.org/project/policyengine-us/
- https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata
- https://github.com/pypa/readme_renderer/blob/main/readme_renderer/__main__.py

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Thank you @westurner! Could you please also
* Run `make format` (we use `black` which requires double quotes)
* Populate `changelog_entry.yaml` as per other recently merged PRs (this can be a `patch`)

**Comment by westurner** [comment]
Should be ready to merge

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- bump: patch
+  changes:
+    changed:
+    - DOC,BLD: setup.py: set `long_description_content_type='text/markdown'` for pypi
```
Comment (nikhilwoodruff) [inline changelog_entry.yaml:4]:
```suggestion
    - Fixed README.md display on PyPI.
```

**Comment by westurner** [comment]
Thanks!

**Pull request merged**